### PR TITLE
[Testing:TAGrading] Rubric Grading Interface

### PR DESCRIPTION
**Pull request by manzil-infinity180**
### Please check if the PR fulfills these requirements:

* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant
* [ ] Screenshots are attached to Github PR if visual/UI changes were made

### What is the current behavior?
Currently no test for rubric/manual grading interface

### What is the new behavior?
Added test for rubric/manual grading interface

### CODE REVIEW

**Comment by manzil-infinity180** [comment]
In future we can verify it throw api test

File: site/cypress/e2e/Cypress-TAGrading/rubric_grading.spec.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        cy.get('[data-testid="component-container"]').eq(0).should('contain', 'Read Me');
+        cy.get('[data-testid="component-container"]').eq(1).should('contain', 'Coding Style');
+        cy.get('[data-testid="component-container"]').eq(2).should('contain', 'Documentation');
+        cy.get('[data-testid="component-container"]').eq(3).should('contain', 'Extra Credit');
+        cy.get('[data-testid="component-64"]').should('contain', 'Read Me');
+        cy.get('[data-testid="component-64"]').click();
+        cy.get('[data-testid="component-64"]')
+            .should('contain', 'Full Credit')
+            .should('contain', 'Minor errors in Read Me')
+            .should('contain', 'Major errors in Read Me or Read Me missing');
```
Comment (IDzyre) [inline site/cypress/e2e/Cypress-TAGrading/rubric_grading.spec.js:18]:
```suggestion
        cy.get('[data-testid="component-64"]')
            .should('contain', 'Full Credit')
            .and('contain', 'Minor errors in Read Me')
            .and('contain', 'Major errors in Read Me or Read Me missing');
```

Same with other uses of chained should calls

**Review by IDzyre — CHANGES_REQUESTED** [review]

File: site/public/templates/grading/Component.twig [hunk 2]
```diff
@@ -64,3 +64,4 @@ Required Input:
     {% block mark_list_footer %}
     {% endblock %}
 </div>
+</div>
```
[the remark this answers is not in the document]
Comment (manzil-infinity180) [reply site/public/templates/grading/Component.twig:67]: sorry ig not , i will change it

**Comment by IDzyre** [comment]
Once you get the tests to pass, i'll do a final review and then merge if it looks good.

**Review by IDzyre — APPROVED** [review]
Looks good to me, once the new test names have passed, I will merge.

**Pull request merged**